### ISSUE TITLE
Invalid card created if Multiple -A identities are specified on deploy/start

### ISSUE THREAD

**Issue by davidkel** [hyperledger/composer#2665]
Tried the following command
`composer network deploy -a f.bna -c PeerAdmin -A admin -S adminpw -A admin2 -S adminpw`

which resulted in a single card being built but the metadata.json is probably not valid
`{"version":1,"userName":["admin","admin2"],"businessNetwork":"digitalproperty-network","enrollmentSecret":["adminpw","adminpw"
]}`

It should create 2 separate cards. Also what happens if I specify the '-f' option ?

**Comment by davidkel** [comment]
A user expected to be able to specify -f multiple times (a reasonable thing to do) the start succeeds but you get an error.
```
composer network start -a digitalproperty-network.bna -c FabricAdmin@devserver          -A admin-org1 -C Admin@org1.example.com-cert.pem -f /tmp/admin-org1.card          -A admin-org2 -C Admin@org1.example.com-cert.pem -f /tmp/admin-org2.card
```
response was
```
TypeError: Path must be a string. Received [ 'Admin@org1.example.com-cert.pem',
  'Admin@org1.example.com-cert.pem' ]
Command failed
```

**Comment by mbwhite** [comment]
Will be resolved by #2753
